### ISSUE TITLE
CoInitialize has not been called

### ISSUE THREAD

**Issue by vfilimonov** [vfilimonov/pyppt#6]
On some machines:
```
com_error: (-2147221008, 'CoInitialize has not been called.', None, None)
```
when using `pyppt_server`

**Comment by vfilimonov** [comment]
Looks like a solution:

https://stackoverflow.com/questions/37258257/why-does-this-script-not-work-with-threading-python

**Comment by vfilimonov** [comment]
Fixed by 76773e9e487593509bbcebe8fcc2166cfcd2f606
